components/apps/ptp-operator/overlays/default/kustomization.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
+  - github.com/redhat-partner-solutions/vse-catalog/components/apps/ptp-operator/overlays/stable?ref=main
+  #- github.com/jnunyez/vse-catalog/components/apps/ptp-operator/overlays/stable?ref=add-ptp
+
+
+patches:
+  - target:
+      kind: PtpOperatorConfig
+      name: default
+    patch: |-
+     - op: replace
+       path: '/spec/daemonNodeSelector'
+       value: {"node-role.kubernetes.io/worker-cnf" : ""}
+     - op: replace
+       path: '/spec/ptpEventConfig/enableEventPublisher'
+       value: true
+     - op: replace
+       path: '/spec/ptpEventConfig/transportHost'
+       value: "amqp://amq-interconnect.openshift-ptp.svc.cluster.local"
+  - target:
+      kind: OperatorGroup
+      namespace: openshift-ptp
+    patch: |-
+     - op: replace
+       path: '/metadata/name'
+       value: 'ptp-operators'
```

Comment:
Should this have been in the other commit I reviewed around PTP?

Reply:
changed the name of the PR.